technic/tools/init.lua

```diff
@@ -10,7 +11,7 @@ if enabled("flashlight")        then dofile(path.."/flashlight.lua") end
 if enabled("cans")              then dofile(path.."/cans.lua") end
 if enabled("chainsaw")          then dofile(path.."/chainsaw.lua") end
 if enabled("tree_tap")          then dofile(path.."/tree_tap.lua") end
-if enabled("sonic_screwdriver") then dofile(path.."/sonic_screwdriver.lua") end
+if enabled("sonic_screwdriver") and mesecons_materials then dofile(path.."/sonic_screwdriver.lua") end
```

Comment:
And this check related to screwdriver / recipe thing (crafting guides should be able to handle missing items):
```suggestion
if enabled("sonic_screwdriver") then dofile(path.."/sonic_screwdriver.lua") end
```

If required crafting guide has issues with this then better to just add check around recipe registration instead.